lasagne/layers/recurrent.py

```diff
@@ -564,9 +550,9 @@ def get_output_for(self, input, mask=None, *args, **kwargs):
         # Because the input is given for all time steps, we can precompute
         # the inputs to the gates before scanning.
         # input is dimshuffled to (n_time_steps, n_batch, n_features)
-        # W_in_to_gates is (n_features, 4*num_units). input_dot_W is then
+        # W_in_to_gates is (n_features, 4*num_units). input_dot_w is then
```

Comment:
Do you want to revert/change whichever commit you made the change to the lowercase w, or do you want me to make another commit where I change it back?  The former is much cleaner but I'm not going to force you to do it! :)